10.hello-docker-compose/README.md

```diff
@@ -121,6 +121,16 @@ altentiavely, you can just use Docker Compose directly:
 docker-compose up
 ```
 
+If everything went well logs from nodeapp about successful persistence of orders shoulod appear:
```

Comment:
nit : fix typo

nit : need better rephrasing , (below is just one example..)

For the successful run , logs from `nodeapp`  should appear in order as follows:

Reply:
Changed it